src/LanguageManager/LanguageEditView.php

```diff
@@ -14,13 +15,20 @@
 	private $languages;
 
 	/**
+	 * @var \Inpsyde\MultilingualPress\Common\Labels
```

Comment:
`Labels` already is imported. Only use FQNs for global stuff, for example, `\WP_Post`.